### PR TITLE
fix typo for structuring in README

### LINKED ISSUE #6: Typo in course schedule
In the README.md the school schedule includes the course lectures, titles, and links to slides. However, one of the links is misspelled. It should be "structuring Python packages" not "strucuring Python packages".

### PR DESCRIPTION
**Pull request by munkm**
I noticed in #6 that there's a typo in the README. This resolves #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Welcome! This is the repository for the 2024 URSSI Summer School on Research Sof
...
-| July 29th, 9:45am-12pm | [Software design](https://evamaxfield.github.io/winter-school-lectures/software-design-and-modularity.slides.html#/), [Structuing Python packages](https://docs.google.com/presentation/d/1thjeqGGnx40pnVobCNxgsXxXA14WimmqlH2tPomOpSg/edit?usp=sharing), [Distributing your Science][Distributing your Science] | Matthew |
+| July 29th, 9:45am-12pm | [Software design](https://evamaxfield.github.io/winter-school-lectures/software-design-and-modularity.slides.html#/), [Structuring Python packages](https://docs.google.com/presentation/d/1thjeqGGnx40pnVobCNxgsXxXA14WimmqlH2tPomOpSg/edit?usp=sharing), [Distributing your Science][Distributing your Science] | Matthew |
 | July 29th, 12-1:00pm | Lunch | |
 | July 29th, 1pm-3pm | [Software design](https://evamaxfield.github.io/winter-school-lectures/software-design-and-modularity.slides.html#/) , [Structuing Python packages](https://docs.google.com/presentation/d/1thjeqGGnx40pnVobCNxgsXxXA14WimmqlH2tPomOpSg/edit?usp=sharing), [Distributing your Science][Distributing your Science] | Matthew |
```
Comment (matthewfeickert) [inline README.md]:
```suggestion
| July 29th, 1pm-3pm | [Software design](https://evamaxfield.github.io/winter-school-lectures/software-design-and-modularity.slides.html#/) , [Structuring Python packages](https://docs.google.com/presentation/d/1thjeqGGnx40pnVobCNxgsXxXA14WimmqlH2tPomOpSg/edit?usp=sharing), [Distributing your Science][Distributing your Science] | Matthew |
```

Just fixing the second instance of the typo here too. :+1:

**Review by matthewfeickert — CHANGES_REQUESTED** [review]
Looks good @munkm, but I think we have one other typo that might have conveniently for pedagogical purposes gotten ignored. :wink: 

Can you also show using the "resolves" syntax to close the issue?

**Review by matthewfeickert — APPROVED** [review]
LGTM. :shipit:

**Comment by munkm** [comment]
Resolves #6

**Pull request merged**